docs/analytics/index.mdx

```diff
@@ -19,9 +19,9 @@ For more details on setting up Sourcegraph Analytics, see our [enablement instru
 
 ## Air-gapped Analytics
```

Comment:
capitalize the product name "Air-Gapped Analytics" for consistency if that's how we are doing this